testapi.pm

```diff
@@ -1791,14 +1791,14 @@ sub data_url($) {
 
 =for stopwords GiB failok OpenQA WebUI
 
-  upload_logs($file [, failok => 0, timeout => 90 ]);
+  upload_logs($file [, failok => 0, timeout => 90, destination_log_name => "custom_name.log" ]);
```

Comment:
It seems we have a mixture in these summary lines (not only this method) mixing "default options", "examples", "possible values". To be at least a bit consistent I recommend to s/"custom_name.log"/$autotest::current_test/

I would appreciate if you update the whole file to be consistent. My suggestion would be to use possible values only in the summary line, e.g. `upload_logs($file [, failok => 0|1, timeout => $timeout, destination_log_name => $destination_log_name ]);`

Reply:
I think you also mixing things here : 
1. adding additional parameter to existing function 
2. improving overall code / documentation quality
While this commit is just about first point you pushing me to do second. 
In anycase I can't understand what you meant by  "timeout => $timeout " ? what it suppose to say ? from string "timeout => 90" I can at least understand that I suppose to bypass timeout as digit and not boolean for example

Reply:
I am not pushing you to anything. I stated: I would appreciate. This is my opinion. But the real issue is that you are not following the existing convention but provide an example where previously it was stating the default value which is is something different. This is why I still suggest to write `$autotest::current_test` instead

Reply:
log_name don't have default value at all,  when log_name is undefined **another** variable  ($upname) will get autogenerated value based on :
```
my $basename = basename($file);
my $upname   = $args{log_name} || ref($autotest::current_test) . '-' . $basename;
```
and defining in documentation "log_name => $log_name"  or even "log_name => $autotest::current_test" will not help to understand this